rpi-eeprom-update

```diff
@@ -193,22 +193,22 @@ checkDependencies() {
    fi
 
    if vcgencmd bootloader_config | grep -qi "Command not registered"; then
-      die "vcgencmd: bootloader_config. not supported. Please update VC firmware"
+      die "vcgencmd: 'bootloader_config` command not supported. Please update VC firmware"
```

Comment:
Is it just me or do those single quotes not match? Shouldn't they just be normal `'` characters?